### PR TITLE
TTAB-23_v2

### PR DESCRIPTION
**Pull request by Joshua-Beets**
sorry had some problems with the branch PR and the fact that i had that on already so i mode a version two and hope that you can just delete the old version and just use this one please let me know

### CODE REVIEW

File: routes/api/index.js [hunk 1]
```diff
@@ -12,4 +13,11 @@ router.route('/test')
     }))
 
 
+    router.route('/timeData')
+    .get((req, res) => res.json({ message: Time }))
+    .post((req, res) => res.json({
+        message: 'test post response successful',
+        body: req.body
+    }))
+
```
Comment (jmclain-banyan) [inline routes/api/index.js:22]: we should be filtering using params in the URL of user id

**Review by jmclain-banyan — CHANGES_REQUESTED** [review]
[{
  "id": 1,
  "date": "9/13/2021",
  "startTime": "10:55 AM",
  "endTime": "10:28 AM"
}, {
  "id": 2,
  "date": "9/15/2021",
  "startTime": "12:50 PM",
  "endTime": "1:10 PM"
}, {
  "id": 3,
  "date": "9/14/2021",
  "startTime": "1:48 PM",
  "endTime": "2:28 PM"
}, {
  "id": 4,
  "date": "9/15/2021",
  "startTime": "11:25 AM",
  "endTime": "2:56 PM"
}, {
  "id": 5,
  "date": "9/13/2021",
  "startTime": "7:41 AM",
  "endTime": "1:44 PM"
}, {
  "id": 6,
  "date": "9/13/2021",
  "startTime": "8:03 AM",
  "endTime": "1:24 PM"
}, {
  "id": 7,
  "date": "9/14/2021",
  "startTime": "11:25 AM",
  "endTime": "10:05 AM"
}, {
  "id": 8,
  "date": "9/13/2021",
  "startTime": "9:41 AM",
  "endTime": "10:36 AM"
}, {
  "id": 9,
  "date": "9/15/2021",
  "startTime": "8:52 AM",
  "endTime": "11:59 AM"
}, {
  "id": 10,
  "date": "9/17/2021",
  "startTime": "1:28 PM",
  "endTime": "10:17 AM"
}, {
  "id": 11,
  "date": "9/13/2021",
  "startTime": "2:34 PM",
  "endTime": "9:06 AM"
}, {
  "id": 12,
  "date": "9/14/2021",
  "startTime": "12:06 PM",
  "endTime": "10:05 AM"
}, {
  "id": 13,
  "date": "9/17/2021",
  "startTime": "11:48 AM",
  "endTime": "10:26 AM"
}, {
  "id": 14,
  "date": "9/15/2021",
  "startTime": "7:58 AM",
  "endTime": "10:21 AM"
}, {
  "id": 15,
  "date": "9/14/2021",
  "startTime": "8:38 AM",
  "endTime": "7:30 AM"
}, {
  "id": 16,
  "date": "9/14/2021",
  "startTime": "7:56 AM",
  "endTime": "10:30 AM"
}, {
  "id": 17,
  "date": "9/13/2021",
  "startTime": "1:59 PM",
  "endTime": "8:21 AM"
}, {
  "id": 18,
  "date": "9/14/2021",
  "startTime": "11:36 AM",
  "endTime": "12:23 PM"
}, {
  "id": 19,
  "date": "9/17/2021",
  "startTime": "1:59 PM",
  "endTime": "2:39 PM"
}, {
  "id": 20,
  "date": "9/13/2021",
  "startTime": "7:46 AM",
  "endTime": "11:08 AM"
}]

**Review by jmclain-banyan — CHANGES_REQUESTED** [review]
Overall looks good😄 just need to resolve merge conflicts

**Review by jmclain-banyan — APPROVED** [review]

**Comment by Joshua-Beets** [comment]
what

**Pull request merged**